README.md

```diff
@@ -40,6 +40,7 @@ OMERO.server configuration.
 
 OMERO system user, group, permissions, and data directory.
 You may need to change these for in-place imports.
+Directory paths must not have a trailing `/`
```

Comment:
This seems quite error prone. What's the behavior if it _does_ have a trailing slash? Is there no filter to strip it conditionally?